website/pages/en/index.js

```diff
@@ -301,8 +301,8 @@ class GetStartedSection extends React.Component {
                   Then add this config to <code>package.json</code>:
                   <MarkdownBlock>
                     {json({
-                      "husky": {
-                        "hooks": {
+                      husky: {
+                        hooks: {
```

Comment:
> No longer a valid JSON?

This file is written in JavaScript, not JSON, so no worries.

Reply:
Indeed, but this is supposed to be readily copy-pasted in `package.json`.

Omitting the double quotes leads to an error message on my machine:
`JSONError: Unexpected token h in JSON`

Reply:
You're right. I now realize there's a formatter involved.

This solves this format discussion.

Shall I squash my commits?

Reply:
No, we use “Squash and merge” for all PRs here so there’s no need to.